docs/source/package-spec.rst

```diff
@@ -89,63 +89,67 @@ name collisions with future versions of the specification.
 
 ----
 
-EthPM Manifest Version: ``manifest_version``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+EthPM Manifest Version: ``manifest``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``manifest_version`` field defines the specification version that
-this document conforms to. Packages **must** include this field.
+The ``manifest`` field defines the specification version that
+this document conforms to.
+
+  - Packages **must** include this field.
 
   :Required: Yes
-  :Key: ``manifest_version``
+  :Key: ``manifest``
   :Type: String
-  :Allowed Values: ``2``
+  :Allowed Values: ``ethpm/3``
 
-.. _package names:
+.. _package-names:
 
 ----
 
-Package Name: ``package_name``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Package Name: ``name``
+~~~~~~~~~~~~~~~~~~~~~~
 
-The ``package_name`` field defines a human readable name for this
-package. Packages **must** include this field. Package names **must**
-begin with a lowercase letter and be comprised of only lowercase
-letters, numeric characters, and the dash character ``-``. Package
-names **must** not exceed 255 characters in length.
+The ``name`` field defines a human readable name for this package.
 
-  :Required: Yes
-  :Key: ``package_name``
+  - Packages **must** include this field to be released on an EthPM registry.
```

Comment:
I'm not sure about this line.  I think i see the motivation here, but I'm conflicted about where the right place for this should be and I don't think it is here.  

| | named | unnamed |
| --- | --- | --- |
| **from registry** | must match | default to name from registry |
| **not from registry** | always valid | user must provide name |

I'd propose going with the logic above which suggests that the name is strictly optional but providing a warning that a package without a name is subject to ambiguous naming.

Reply:
Gotcha, agreed this seems like a neater way of handling this.